m4/nx-xtrans.m4

```diff
@@ -0,0 +1,168 @@
+dnl
```

Comment:
> nx copyright is missing here

yes and no. The file is really a plain copy of the Xtrans code. I don't consider plainly copying a file (and renaming the functions) good reason enough to add us as copyright holders.